src/vma/sock/sock-redirect.cpp

```diff
@@ -776,6 +819,9 @@ int setsockopt(int __fd, int __level, int __optname,
 		ret = p_socket_object->setsockopt(__level, __optname, __optval, __optlen);
 	}
 	else {
+		BULLSEYE_EXCLUDE_BLOCK_START
```

Comment:
Here you are delaying the call to get_orig_funcs() just before needing orig_os_api.setsockopt, which is also OK.
However need to keep the same code style (either calling at startup or just before needing which is the preferred style in my view).
Please rewrite keeping the same code style